exercises/sgf-parsing/SgfParsingTest.fs

```diff
@@ -1,98 +1,63 @@
-// This file was created manually and its version is 1.0.0.
+// This file was auto-generated based on version 1.0.0 of the canonical data.
 
 module SgfParsingTest
 
-open Xunit
 open FsUnit.Xunit
+open Xunit
 
 open SgfParsing
 
-// This exercises requires you to do somewhat more complicated parsing.
-// This is the perfect exercise to get to know the FParsec library,
-// as the FParsec library helps you define parsers in a very convenient way.
-// You can find the FParsec documentation here: http://www.quanttec.com/fparsec/
-
-// A tree consists of two parts:
-// - The node's data: a map of type Map<string, string list> 
-// - A list of children (which are itself trees)
-// You can define the data type as follows (but feel free to define your own):
-// type Data = Map<string, string list>
-// type Tree = Node of Data * Tree list
-
-let treeWithChildren node children = mkTree (Map.ofList node) children
-let treeWithSingleChild node child = mkTree (Map.ofList node) [child]
-let treeWithNoChildren node = mkTree (Map.ofList node) []
-
 [<Fact>]
-let ``Empty value`` () =
-    let input = "" 
-    let expected = None
-    parseSgf input |> should equal expected
+let ``Empty input`` () =
+    parseSgf "" |> should equal None
 
 [<Fact(Skip = "Remove to run test")>]
-let ``Tree without nodes`` () =
-    let input = "()"
-    let expected = None
-    parseSgf input |> should equal expected
+let ``Tree with no nodes`` () =
+    parseSgf "()" |> should equal None
 
 [<Fact(Skip = "Remove to run test")>]
 let ``Node without tree`` () =
-    let input = ";"
-    let expected = None
-    parseSgf input |> should equal expected
+    parseSgf ";" |> should equal None
 
 [<Fact(Skip = "Remove to run test")>]
 let ``Node without properties`` () =
-    let input = "(;)"
-    let expected = Some (treeWithNoChildren [])
-    parseSgf input |> should equal expected
+    let expected = Some (Node (Map.empty, []))
+    parseSgf "(;)" |> should equal expected   
 
 [<Fact(Skip = "Remove to run test")>]
 let ``Single node tree`` () =
-    let input = "(;A[B])"
-    let expected = Some (treeWithNoChildren [("A", ["B"])])
-    parseSgf input |> should equal expected
+    let expected = Some (Node (Map.empty.Add("A", ["B"]), []))
```

Comment:
I think I would prefer to use `Map.ofList` instead of `Map.empty.Add`.
```suggestion
    let expected = Some (Node (Map.ofList [("A", ["B"])], []))
```